src/OidcClient.php

```diff
@@ -192,7 +192,8 @@ public function generateAuthorizationRedirect(?string $prompt = NULL, array $sco
       $data['prompt'] = $prompt;
     }
 
-    return new RedirectResponse(sprintf('%s?%s', $this->getAuthorizationEndpoint(), http_build_query($data)));
+    $query = parse_url($this->getAuthorizationEndpoint(), PHP_URL_QUERY);
+    return new RedirectResponse(sprintf('%s%s%s', $this->getAuthorizationEndpoint(), ($query) ? '&' : '?', http_build_query($data)));
```

Comment:
Alright, but I believe it would be better to parse the authorization endpoint into a array object, and to combine it with the data object. Something like:

```suggestion
    // Retrieve possible query parameters from the configured authorization endpoint
    $endpointQueryData = [];
    parse_str(parse_url($this->getAuthorizationEndpoint(), PHP_URL_QUERY), $endpointQueryData));
    $data = array_merge($endpointQueryData, $data);

    return new RedirectResponse(sprintf('%s?%s', $this->getAuthorizationEndpoint(), http_build_query($data)));
```